source/system/Numbers.ooc

```diff
@@ -100,7 +100,7 @@ LDouble: cover from long double {
 		result
 	}
 	in: func (range: Range) -> Bool {
-		this >= range min && this < range max
+		this as Int >= range min && this as Int < range max
```

Comment:
The question here I guess is if the comparison should be `... this as Int <= range max` rather than `... this as Int < range max`. I leave it to @marcusnaslund to decide